packages/components/src/DataList/DataList.types.ts

```diff
@@ -350,6 +350,11 @@ export interface DataListActionProps<T extends DataListObject> {
    * The callback function when the action is clicked.
    */
   readonly onClick?: (data: T) => void;
+
+  /**
+   * The URL to navigate to when the action is clicked.
+   */
+  readonly actionUrl?: string;
 }
```

Comment:
Oh that's a great idea! Yes we do have some _undesirable_ XOR happenings 😆  but I fully agree that this would be a great use of it.  I can absolutely see confusion happening around the `onClick` & `actionUrl` trying to be used at the same time.  I'll update this!